### ISSUE TITLE
Improve the text field

### ISSUE THREAD

**Issue by sseppi** [noi-techpark/it.bz.opendatahub.epaper.webapp#88]
As a data editor I want that all text field:
- in case of need, wraps the text in a multiline accordingly to the horizontal space available in order to be able to see the entire title if is longer than a row;
- is associated to a maximum height in order to avoid overlapping;
- if the text is too long is automatically cut (three dots should be added at the end to let the user understand that the text has been cut) in order to not exceed the maximum width and height available.

**Comment by SaimonasFOS** [comment]
> is associated to a maximum height in order to avoid overlapping;

@sseppi Does this mean that user needs to have an ability to specify maximum height together with other text field properties (text position, font size, etc..)? It does not seem clear from this sentence alone how the maximum height would be determined.

**Comment by sseppi** [comment]
@SaimonasFOS Yes, for this iteration I would give to the user the possibility to specify maximum height together with other text field properties (text position, font size, etc..) so the user will be the one that avoid overlapping between two fields.

**Comment by sseppi** [comment]
@SaimonasFOS I've seen that the fiels custom text doesn't wrap on multiple lines. Qhóuld it be possible to add the same feature as the other fields also to this one or is it too complex?

**Comment by svaickus** [comment]
@sseppi Custom text fields should wrap as well, only problem is that you cannot see it immediately on the image preview. Try to save the fields and check the actual preview in the "Information" tab of the display (or press "Show Content" if you are editing a template instead).
If the text still doesn't wrap check if the field "height" property is big enough.

**Comment by sseppi** [comment]
@svaickus now it works! Thanks
